docker-compose.dev.yml

```diff
@@ -55,7 +35,6 @@ services:
       - postgres_data_dev:/var/lib/postgresql/data
     networks:
       - activepieces_dev
-#    command: -c log_statement=all -c shared_preload_libraries=auto_explain -c auto_explain.log_min_duration=0
 
```

Comment:
Can you please revert this line? It's helpful in debugging Postgres queries, but it produces a lot of logs that's why it's commented by default :sweat_smile: